index.html

```diff
@@ -0,0 +1,44 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Survey Form</title>
+    <link rel="stylesheet" href="stylesheet.css">
+</head>
+
+<body>
+    <h1 id="title">Survey Form</h1>
+    <p id="description"> Survey carried out in 2020</p><br>
+    <form id="survey-form">
+        <input list="dropdown">
+        <datalist id="dropdown">
+            <option value="1-4">
+            <option value="1-4">
+            <option value="1-4">
+        </datalist><br><br>
+        <label for="male">Male</label>
+        <input type="radio" name="gender" id="male" value="male"><br>
+        <label for="female">Female</label>
+        <input type="radio" name="gender" id="female" value="female"><br>
+        <label for="other">Other</label>
+        <input type="radio" name="gender"id="other" value="other"><br><br>
+        <label id="name-label" for="name">Name:</label>
+        <input id="name" type="text" name="name" placeholder="Your Name" required><br>
```

Comment:
#### `AWESOME`


Good job associating the `input` element with the `label` element using `id` and `for` attribute respectively. :clap: :+1: 

> #### Associating a `<label>` with an `<input>` element offers some major advantages:
> 
> - The label text is not only visually associated with its corresponding text input; it is programmatically associated with it too. This means that, for example, a screen reader will read out the label when the user is focused on the form input, making it easier for an assistive technology user to understand what data should be entered.
> 
> - You can click the associated label to focus/activate the input, as well as the input itself. This increased hit area provides an advantage to anyone trying to activate the input, including those using a touch-screen device.

An Alternative way to associate them is to wrap the label element with the `<input>` element. 

- Example of nesting the `<input>` directly inside the `<label>` : 

``` html
  <label id="name-label" for="name">Name: <br>
       <input type="text" name="name" id="name" placeholder="Your Name" required>
  </label> 
```

You can learn more [here](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/label)